go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1036,7 +1036,7 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 		// TODO(adrianc): Revisit above comment
 		// TODO(jtanenba): LocalPodInformer informs on all pods
 		nodeController, err := honode.NewNode(
-			nc.Kube,
+			nil,
```

Comment:
Can we do something different here? How is it this being nil intrinsic to what type of controller we want to run?

Reply:
Yeah, I can add a new argument here. 
When `NewNode` is used by ovnkube-node process, the arg `kube` is not used anyway. So I thought it could be used as an indicator for choosing what kind of controller we want to run.

Reply:
fixed